### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by william-stacken**
Ignore all the commits, they are from feature/presentation and have already been merged into dev. My mistake.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ - **GitHub Actions**
+
+## 3 Dependencies
+The following _must_ be installed:
+ - **JDK 11**: Needed to compile the project.
+
+_Either_ of the following _must_ be installed:
+ - **Visual Studio Code**: Can be used to compile and run tests using
+                           its [java extension](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack).
+ - **maven**: ?
```
Comment (william-stacken) [inline README.md]: @FilipKTH would you like to share how you compile the application or should we skip it?

Comment (fiilp) [reply README.md]: I use the standard maven commands to run the jar and run the tests. It's also independent of the IDE:

**Tests**
`mvn test`
**Run**
`mvn spring-boot:run`

Comment (william-stacken) [reply README.md]: Fixed! Is this branch ready for merge?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- Constants are named with **capitalized snake_case**. (ex. THIS_IS_A_CONSTANT)
+- Opening brackets are placed on a new line for classes, but not
+  for `for`, `while`, `if` blocks, and functions.
+- Brackets are not required for `if`, `while` and `for` loops containing
+  just one line of code.
+- Tab size is 4 spaces.
+
+## 5. Testing
+A feature branches must be properly tested before their PRs can be merged. In general, there
+must be a test source file for each source file, and it should test both invalid and valid inputs
```
Comment (Cactu5) [inline CONTRIBUTING.md]: "... a test source file for each source file ..." seems too specific, maybe something like "Test all public methods, except for simple getters and setters."?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## 1. Repository Structure
+The recruitment application is a maven project and thus uses the
+following repository structure.
+ - **src/main/java**: Contains source files, organized in layers following
+                      domain-driven design.
+ - **src/main/resources**: Contains resources such as thymeleaf templates and
+                           properties for loggers, internationalization, etc.
+ - **src/test**: Contains Junit tests for each source file, making use of
+                      spring's mockMVC to simulate HTTP transactions.
```
Comment (Cactu5) [inline README.md]: I think "each source file" is a bit too much, maybe something like "Test all public methods, except for simple getters and setters."?

**Review by Cactu5 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (william-stacken) [reply CONTRIBUTING.md]: It has been done, opinions?

**Review by Cactu5 — APPROVED** [review]

**Pull request merged**